README.md

```diff
@@ -9,7 +9,17 @@ We'd love to see you contributing to our projects, please just follow a few conv
 - No trailing whitespace. Blank lines should not have any spaces.
 - Do not indent after private/protected.
 - Use Ruby >= 1.9 syntax for hashes. Prefer `{ a: :b }` over `{ :a => :b }`.
-- Prefer `&&`/`||` over and/or.
+- Prefer `&&`/`||` over `and`/`or`.
+- Use `and`/`or` only for the early returns:
+``` ruby
+def some_method_with_and
+  head 401 and return
+end
+
+def some_method_with_or(user)
+  user.admin? or raise UnauthorizedRequestError
```

Comment:
still can't understand how this is better than
```
raise UnauthorizedRequestError unless user.admin?
```
plus its pretty important to not use those with `=` assignments of any kind, as this gives really unexpected results, see https://stackoverflow.com/questions/1426826/difference-between-and-and-in-ruby